src/MusicProcessing.jl

```diff
@@ -37,7 +37,7 @@ export melspectrogram, mfcc
 export spectrogram, stft, istft, phase_vocoder
 
 function __init__()
-    @require Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80" include("display.jl")
+    @require GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a" include("display.jl")
```

Comment:
`GLMakie` -> `Makie`

Reply:
Should I use Makie instead of GLMakie?

Reply:
okay for now use GLMakie. We will resolve this later.